docs/ds4.browser.json

```diff
@@ -3,7 +3,7 @@
         "../src/less/less/ds4/variables.less",
         "../src/less/less/ds4/mixins.less",
         "src/less/ds4/index.less",
-        "./src/js/base/util.js",
+        "require-run: nodelist-foreach-polyfill",
```

Comment:
A nitpick, but could we move this down to where the other polyfill is listed, to keep the requires consistent? Or, does it need to run before the `main.js` files?

Reply:
I believe it needs to be before. To be honest, the details one should also be before.

Reply:
Sounds good. Maybe move the details one up then.